chatkit-android/src/main/kotlin/com/pusher/chatkit/AndroidChatkitDependencies.kt

```diff
@@ -20,6 +23,7 @@ data class AndroidChatkitDependencies @JvmOverloads constructor(
 
     override val mediaTypeResolver: MediaTypeResolver = platformDependencies.mediaTypeResolver
     override val sdkInfo: SdkInfo = chatkitSdkInfo
+    override val pushNotifications: PushNotifications? = BeamsPushNotifications(context)
```

Comment:
This is nullable with null checks on access, but it's always assigned?

Reply:
Does that mean we plan to have it unconditionally as a dependency? If so, I'll just go round and remove the nullability

Reply:
Remove the nullability. And `context` needs to nullable, otherwise it's a breaking change. And we need a runtime check to see if it has been given if notifications are being called.